book/index.qmd

```diff
@@ -35,9 +36,19 @@ predictions
 predictions$score(msr("classif.acc"))
 ```
 
-Here we have picked the 'penguins' task (which is `r mlr3` language for dataset), randomly split the task into 67% training data and 33% testing data, trained a random forest on the training data to learn the probability of an observation falling into one of the outcome classes, showed the fitted model, and then made prediction on the test data, showed these predictions and evaluated the model using the accuracy measure.
+We
 
-Whilst mlr3 makes training and predicting easy, it also uses a unified interface to perform some very complex operations in just a few lines of code:
+* load the `r mlr3` library,
+* choose the 'penguins' "task" (the `r mlr3` term for a dataset),
+* randomly split the task into 67% training data and 33% testing data (this is the default when using the `partition()` function),
+* choose the `rpart` decision tree learner to perform classification,
+* train the model on the task, selecting the training rows from the split,
+* show the fitted model,
+* make predictions for test rows in the penguins task,
+* show the predictions, and
+* compute the accuracy performance measure for those predictions, i.e. the fraction of predictions that are correct.
```

Comment:
Personally I'm not sure the list works here as the point I want to make is that we can do easy things and complex things - but when we write the above as a long list it makes it seem more difficult.